cmake/modules/AddSwift.cmake

```diff
@@ -1352,23 +1363,17 @@ function(add_swift_library name)
               ${SWIFTLIB_SWIFT_MODULE_DEPENDS_LINUX})
         endif()
 
+        # Swift compiles depend on swift modules, while links depend on
+        # linked libraries.  Find targets for both of these here.
         set(swiftlib_module_dependency_targets)
+        set(swiftlib_private_link_libraries_targets)
```

Comment:
Unless I am missing something, isn't the variable swiftlib_private_link_libraries_targets dead?